Dockerfile.test-infra

```diff
@@ -2,9 +2,13 @@ FROM quay.io/ocpmetal/assisted-service:latest AS service
 
 FROM docker.io/centos:8
 
-RUN yum -y install make python3 libvirt-client libvirt-devel gcc unzip wget curl git python3-devel && yum clean all;
+RUN yum -y install make python3 libvirt-client libvirt-devel gcc unzip wget curl git python3-devel yum-utils
```

Comment:
Why did you remove `yum clean all`?

Reply:
I moved it to the end of the installation commands, after the installation of docker-ce and containerd.io. Line 9.